CMakeLists.txt

```diff
@@ -343,6 +343,10 @@ endif()
 # Generate .hipVersion
 file(WRITE "${PROJECT_BINARY_DIR}/.hipVersion" ${_versionInfo})
 
+# Generate hipVersion.h
+set(_versionInfoHeader "//Auto-generated by cmake\n\n#define HIP_VERSION_MAJOR ${HIP_VERSION_MAJOR}\n#define HIP_VERSION_MINOR ${HIP_VERSION_MINOR}\n#define HIP_VERSION_PATCH ${HIP_VERSION_GITDATE}\n")
```

Comment:
```c++
#define HIP_VERSION_PATCH ${HIP_VERSION_GITDATE}
```

contradicts to CMakeLists.txt:

```cmake
set(HIP_VERSION_PATCH ${HIP_VERSION_GITDATE}-${HIP_VERSION_GITHASH})
```

Reply:
hipcc passes only git workweek to application hence we have kept it to maintain current behavior,